Documentation/Settings.md

```diff
@@ -0,0 +1,302 @@
+
+On device help text:
+
+O=off | S=soldering temp | Z=standby at sleep temp until moved | R=standby without heating until moved
+
+### Setting: Cooldown blink
+
+If the idle screen should blink the tip temperature for attention when the device tip is over 50C. Intended as a 'tip is hot' warning.
+
+On device help text:
+
+Flash the temperature reading after heating was halted while the tip is still hot
+
+### Setting: Calibrate temperature?
+
+Used to calibrate the ADC+Op-amp offsets for the tip. This calibration must be performed when the tip temperature and the handle temperature are equal. Generally not required unless your device is reading more than 5C off target.
+
+On device help text:
+
+Start tip temperature offset calibration
+
+### Setting: Factory reset?
+
+Resets all settings and calibrations to factory defaults. Does NOT erase custom user bootup logo's
+
+On device help text:
+
+Reset settings to default
+
+### Setting: Calibrate input voltage?
+
+Enters an adjustment mode where you can gradually adjust the measured voltage to compensate for any unit-to-unit variance in the voltage sense resistors
+
+On device help text:
+
+Start VIN calibration (long press to exit)
+
+### Setting: Detailed solder screen
+
+Should the device show an 'advanced' soldering view. This is a text based view that shows more information at the cost of no nice graphics.
+
+On device help text:
+
+Display detailed information in a smaller font on soldering screen
+
+### Setting: Scrolling speed
+
+How fast the description text scrolls when hovering on a menu. Faster speeds may induce tearing but allow reading the whole description faster.
+
+On device help text:
+
+Speed info text scrolls past at (S=slow | F=fast)
+
+### Setting: QC voltage
+
+This adjusts the maximum voltage the QC negotiation will adjust to. Does NOT affect USB-PD. Should be set safely based on the current rating of your power supply.
+
+On device help text:
+
+Max QC voltage the iron should negotiate for
+
+### Setting: PD timeout
+
+How long until firmware stops trying to negotiate for USB-PD and tries QC instead. Longer times may help dodgy/old PD adapters, faster times move onto PD quickly. Units of 100ms. Reccomended to keep small.
+
+On device help text:
+
+PD negotiation timeout in 100ms steps for compatibility with some QC chargers
+
+### Setting: Power limit
+
+Allows setting a custom Wattage for the iron to aim to keep the AVERAGE power below. The Iron cant control its peak power no matter how you set this. (Except for MHP30 which will regulate nicely to this). If USB-PD is in use, the limit will be set to the lower of this and the supplies advertised wattage.
+
+On device help text:
+
+Maximum power the iron can use (W=watt)
+
+### Setting: Reverse + - keys
+
+Swaps which button increments and decrements the temperature on temperature change screens.
```

Comment:
screens -> screen

Reply:
Well there is more than one of them so.. ?